### PR TITLE
Delay node update information (parent / child setup)

### PR DESCRIPTION
**Pull request by stelfrag**
##### Summary
Currently when a child connects, there is a message that is sent to the cloud to indicate the child is live. Similarly on child disconnection, a message indicating the child has disconnected is sent. 

This PR will delay a bit to make sure the child will not immediately disconnect and avoid sending too many messages to the cloud. 

##### Test Plan
- Setup a parent (claimed to the cloud) with at least one child (activate aclk conversation logs)
   `conversation log = yes` in the `[cloud]` section
- Start the parent and then the child / children
   - Stop and restart the child -- notice disconnect and reconnect messages sent to the cloud
   - Look for `OUTGOING Msg:"UpdateNodeInstanceConnection"` messages in the `aclk.log` 
- Apply the PR and repeat
  - Notice less messages sent to the cloud as you stop and restart a child

### CODE REVIEW

File: streaming/receiver.c [hunk 1]
```diff
@@ -719,10 +722,12 @@ static int rrdpush_receive(struct receiver_state *rpt)
     rrdcontext_host_child_disconnected(rpt->host);
 
 #ifdef ENABLE_ACLK
-    // in case we have cloud connection we inform cloud
-    // new child connected
-    if (netdata_cloud_setting)
-        aclk_host_state_update(rpt->host, 0);
+    // Child disconnection
+    if (netdata_cloud_setting) {
+        struct aclk_database_worker_config *wc = (struct aclk_database_worker_config *)rpt->host->dbsync_worker;
```
Comment (vlvkobal) [inline streaming/receiver.c:727]: Why do you cast the type explicitly? I don't think you need it. Moreover, you didn't do that a few lines before.

File: streaming/receiver.c [hunk 2]
```diff
@@ -719,10 +722,12 @@ static int rrdpush_receive(struct receiver_state *rpt)
...
+        if (!wc || wc->update_node_after == -1)
+            wc->update_node_after = now_realtime_sec() + ACLK_NODE_UPDATE_DELAY * 3;
```
Comment (vlvkobal) [inline streaming/receiver.c:729]: If `wc` is `NULL` (the first part of the condition) we can't dereference it.

**Comment by thiagoftsm** [comment]
Except for comments that @vlvkobal  wrote, the tests show PR working as expected, please, @stelfrag address them.

File: streaming/receiver.c (hunk 1 above)
Comment (stelfrag) [reply streaming/receiver.c:727]: Will update

File: streaming/receiver.c (hunk 2 above)
Comment (stelfrag) [reply streaming/receiver.c:729]: What was I thinking.. :smiley:  will fix

**Review by vlvkobal — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]
I ran a new round of tests  with latest commit, lgtm!

**Pull request closed without merging**